mcstatus/protocol/connection.py

```diff
@@ -448,6 +448,9 @@ def __init__(self) -> None:
 
     def read(self, length: int) -> bytearray:
         """Return :attr:`.received` up to length bytes, then cut received up to that point."""
+        if len(self.received) == 0:
```

Comment:
Is `== 0` the right check to make here? Wouldn't `< length` make more sense?

Reply:
Fixed [here](https://github.com/py-mine/mcstatus/compare/7a22f5040d76f759ebb580ce719740da4ed0cf74..9c4af29f244827dc306cf1f9277f8b27d896abb4), thanks!